L1Trigger/L1TTwinMux/src/DTLowQMatching.cc

```diff
@@ -41,11 +41,14 @@ void DTLowQMatching::Matching(int track_seg){
 
   L1MuDTChambPhDigi * dtts=0;
   L1MuDTChambPhDigi * rpcts1=0;
-  std::auto_ptr<L1MuTMChambPhContainer> m_phiRPCDigis_tm(new L1MuTMChambPhContainer);
+  std::shared_ptr<L1MuTMChambPhContainer> m_phiRPCDigis_tm(new L1MuTMChambPhContainer);
+   //auto m_phiRPCDigis_tm=std::make_shared<L1MuTMChambPhContainer()>;
```

Comment:
Hi @gkaratha what was the problem with this line?
It looks as if just the brackets were at the wrong place:
`auto m_phiRPCDigis_tm=std::make_shared<L1MuTMChambPhContainer>();`